api/tools/build.gradle

```diff
@@ -86,8 +86,15 @@ sourceSets {
                   // In general, command-line tools won't need access to
                   // org.pmiops.workbench.model (Swagger-generated classes),
                   // so we selectively whitelist as needed.
+                  'org/pmiops/workbench/auth/Constants.java',
```

Comment:
Rant mode: dealing with this per-file dependency management was a nightmare. I can't wait until we get rid of this, it will make it 5x easier to write small command-line tools.

Reply:
Can we bump that Tech Debt ticket up? It sounds like it could be really valuable to deal with this before we start getting into more command line overrides and breaking up our command line tools.

Reply:
Agreed. This is https://precisionmedicineinitiative.atlassian.net/browse/RW-2172

Reply:
+1, though I'm a bit confused - why did you define the tool here rather than using your new dependency approach? (defining the tool at the api/build.gradle level?)  I imagined we might move towards eliminating this gradle file entirely at some point.